backend/web/src/main/java/com/woowacourse/taggle/tag/controller/TagController.java

```diff
@@ -51,6 +51,14 @@
                 .body(tagBookmarkResponse);
     }
 
+    @GetMapping("/untagged")
+    public ResponseEntity<TagBookmarkResponse> findUntagged(@AuthenticationPrincipal final SessionUser user) {
```

Comment:
```suggestion
    public ResponseEntity<TagBookmarkResponse> findUntaggedBookmarks(@AuthenticationPrincipal final SessionUser user) {
```
이렇게 바꾸는걸 제안하려고 보니 차라리 BookmarkController에 두고 `bookmarks/untagged` URI를 가지는게 더 자연스러운 것 같은데 어떻게 생각하시나요?

Reply:
결정이 쉽지가 않네요 한번 같이 이야기해봐도 좋을거 같아요